stories/core/LabeledInput.stories.tsx

```diff
@@ -20,6 +20,9 @@ export default {
     disabled: { type: 'boolean' },
     required: { type: 'boolean' },
   },
+  args: {
+    size: undefined,
```

Comment:
Why it is needed?

Reply:
To explicitly enable the prop. Otherwise it doesn't show up (since it's inherited from `Input`).

Reply:
Restarted storybook and it works without args. So removed